<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <link rel="shortcut icon" href="https://static.igem.wiki/common/icons/favicons/igem-2022.svg" type="image/x-icon">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Model | IIT-Bombay - iGEM 2024</title>
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.6/dist/css/bootstrap.min.css" rel="stylesheet"
    integrity="sha384-4Q6Gf2aSP4eDXB8Miphtr37CMZZQ5oXLH2yaXMJ2w8e2ZtHTl7GptT4jmndRuHDT" crossorigin="anonymous">
  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.6/dist/js/bootstrap.bundle.min.js"
    integrity="sha384-j1CDi7MgGQ12Z7Qab0qlWQ/Qqz24Gc6BM0thvEMVjHnfYGF0rmFCozFSxQBxwHKO"
    crossorigin="anonymous"></script><link rel="stylesheet" href="style.css">
  </head>

<body>
    <!-- <div id="root">
        <div
            style="position: fixed; top: 0px; left: 0px; width: 0%; height: 5px; background-color: green; z-index: 9999; transition: width 0.25s;">
        </div> -->
        <div class="container">
            <nav class="navbar navbar-expand-lg bg-body-tertiary">
        
              <a class="tab-link pic" id="home-logo" href="/iit-bombay/">
                <!-- <img src="https://static.igem.wiki/teams/5365/wiki-assets/nav/calcicapture.svg"></a> -->
              <div class="container-fluid">
                <div class="menu-item">
                  <a href="index.html" class="btn btn-outline-* custom-home-btn">Home</a>
                </div>
                <div class="dropdown">
                  <button class="btn btn-outline-* dropdown-toggle" type="button" data-bs-toggle="dropdown"
                    aria-expanded="false" style="color:rgb(0, 0, 0);">
                    Team
                  </button>
                  <ul class="dropdown-menu">
                    <li><a class="dropdown-item" href="Team.html">Team</a></li>
                    <li><a class="dropdown-item" href="Attributes.html">Attributes</a></li>
                  </ul>
                </div>
                <div class="dropdown">
                  <button class="btn btn-outline-* dropdown-toggle" type="button" data-bs-toggle="dropdown"
                    aria-expanded="false" style="color: rgb(0, 0, 0);">
                    Project
                  </button>
                  <ul class="dropdown-menu">
                    <li><a class="dropdown-item" href="Contributions.html">Contribution</a></li>
                    <li><a class="dropdown-item" href="Description.html">Description</a></li>
                    <li><a class="dropdown-item" href="Engineering.html">Engineering</a></li>
                    <li><a class="dropdown-item" href="Notebook.html">Notebook</a></li>
                    <li><a class="dropdown-item" href="Safety.html">Safety</a></li>
                  </ul>
                </div>
                <div class="menu-item"><a href="human-practices.html" class="btn btn-outline-* custom-human-btn">Human practices</a>
                </div>
                <div class="menu-item"><a href="Education.html" class="btn btn-outline-* custom-human-btn">Education</a>
                </div>
                <div class="menu-item"><a href="Model.html" class="btn btn-outline-* custom-human-btn">Model</a>
                </div>
              </div>
          </div>
          </nav>
          </div>
        <div class="contain">
            <main style="display: flex; margin-bottom: 1rem;"><button class="menu-toggle">☰</button>
                <div class="menu-container  ">
                    <div class="menu-sticky">
                        <div class="list-group"><a href="#section-0"
                                class="list-group-item list-group-item-action menu-item">Calcification at cellular
                                Level</a>
                            <div class="sub-items "><a href="#section-0-subsection-0-0"
                                    class="list-group-item list-group-item-action sub-item menu-item">Design</a><a
                                    href="#section-0-subsection-0-1"
                                    class="list-group-item list-group-item-action sub-item menu-item">Build</a><a
                                    href="#section-0-subsection-0-2"
                                    class="list-group-item list-group-item-action sub-item menu-item">Test</a><a
                                    href="#section-0-subsection-0-3"
                                    class="list-group-item list-group-item-action sub-item menu-item">Build</a></div><a
                                href="#section-1" class="list-group-item list-group-item-action menu-item">Population
                                Model of Emilania Huxleyi</a>
                            <div class="sub-items "><a href="#section-1-subsection-1-0"
                                    class="list-group-item list-group-item-action sub-item menu-item">Design</a><a
                                    href="#section-1-subsection-1-1"
                                    class="list-group-item list-group-item-action sub-item menu-item">Build</a><a
                                    href="#section-1-subsection-1-2"
                                    class="list-group-item list-group-item-action sub-item menu-item">Test</a><a
                                    href="#section-1-subsection-1-3"
                                    class="list-group-item list-group-item-action sub-item menu-item">Learn</a></div><a
                                href="#section-2" class="list-group-item list-group-item-action menu-item">PCR
                                Amplification of Calmodulin</a>
                            <div class="sub-items "></div><a href="#section-3"
                                class="list-group-item list-group-item-action menu-item">Cloning of Calmodulin in
                                pET28a</a>
                            <div class="sub-items "><a href="#section-3-subsection-3-0"
                                    class="list-group-item list-group-item-action sub-item menu-item">Cycle 1:
                                    Restriction digestion</a><a href="#section-3-subsection-3-1"
                                    class="list-group-item list-group-item-action sub-item menu-item">Cycle 2: Cloning
                                    of Calmodulin gene using Infusion Kit</a></div><a href="#section-4"
                                class="list-group-item list-group-item-action menu-item">Check the expression of the
                                Calmodulin protein </a>
                            <div class="sub-items "><a href="#section-4-subsection-4-0"
                                    class="list-group-item list-group-item-action sub-item menu-item">Cycle 1:
                                    Expression in BL21-DE3 RIPL cells</a><a href="#section-4-subsection-4-1"
                                    class="list-group-item list-group-item-action sub-item menu-item">Cycle 2: Western
                                    blot to check the Expression in BL21-DE3 RIPL cells</a><a
                                    href="#section-4-subsection-4-2"
                                    class="list-group-item list-group-item-action sub-item menu-item">Cycle 1:
                                    Expression in BL21-DE3 Rosetta cells</a><a href="#section-4-subsection-4-3"
                                    class="list-group-item list-group-item-action sub-item menu-item">Cycle 2: Western
                                    blot to check the Expression in BL21-DE3 Rosetta cells</a></div><a href="#section-5"
                                class="list-group-item list-group-item-action menu-item">Purification of protein from
                                large culture volume</a>
                            <div class="sub-items "><a href="#section-5-subsection-5-0"
                                    class="list-group-item list-group-item-action sub-item menu-item">Affinity
                                    purification</a><a href="#section-5-subsection-5-1"
                                    class="list-group-item list-group-item-action sub-item menu-item">Size exclusion
                                    chromatography purification</a></div><a href="#section-6"
                                class="list-group-item list-group-item-action menu-item">Biophysical Characterization of
                                Purified Calmodulin</a>
                            <div class="sub-items "><a href="#section-6-subsection-6-0"
                                    class="list-group-item list-group-item-action sub-item menu-item">I) CD of
                                    Calmodulin</a><a href="#section-6-subsection-6-1"
                                    class="list-group-item list-group-item-action sub-item menu-item">II)
                                    Fluorescence</a></div><a href="#section-7"
                                class="list-group-item list-group-item-action menu-item">Biochemical Characterization of
                                Calmodulin</a>
                            <div class="sub-items "><a href="#section-7-subsection-7-0"
                                    class="list-group-item list-group-item-action sub-item menu-item">Check the affinity
                                    of Calmodulin with Ca2+ ions</a></div><a href="#section-8"
                                class="list-group-item list-group-item-action menu-item">NAD+ Kinase Assay</a>
                            <div class="sub-items "><a href="#section-8-subsection-8-0"
                                    class="list-group-item list-group-item-action sub-item menu-item">Design</a></div><a
                                href="#section-9" class="list-group-item list-group-item-action menu-item">Cloning of
                                Calmodulin with C-terminal Flag Tag</a>
                            <div class="sub-items "><a href="#section-9-subsection-9-0"
                                    class="list-group-item list-group-item-action sub-item menu-item">Design</a></div><a
                                href="#section-10"
                                class="list-group-item list-group-item-action menu-item">Co-Immunoprecipitation of
                                Interacting Proteins from E. huxleyi</a>
                            <div class="sub-items "><a href="#section-10-subsection-10-0"
                                    class="list-group-item list-group-item-action sub-item menu-item">Design</a></div>
                        </div>
                    </div>
                </div>
                <div class="content-outer">
                    <h1>Engineering</h1>
                    <div class="content-container">
                        <div class="content-container">
                            <section id="section-0">
                                <h2>Calcification at cellular Level</h2>
                                <h3 id="section-0-subsection-0-0">Design</h3>
                                <p>To understand calcification pathway right from uptake to deposition, we decided to
                                    implement it by taking inspiration of different parts of the pathway from different
                                    analogous processes. The process was designed taking into account uptake of calcium
                                    ions, buffering by calmodulin protein, formation of coccoliths inside CV and
                                    deposition on the GPA protein layer around the cell . It was designed in order to
                                    understand the change in overall calcification and thus carbon capture based on
                                    changes in external calcium concentration, ambient (non-limiting) deposition
                                    environment and buffering protein- calmodulin.</p>
                                <h3 id="section-0-subsection-0-1">Build</h3>
                                <p>The process built took into account uptake across the membrane whose transmission
                                    rate depends on concentration gradient and activation of uptake channels by free
                                    calmodulin molecules. Apart from the calcium used for the cytosolic functions, the
                                    rest is bound to free calmodulin. The bound calmodulin is unloaded at CV inside
                                    which the formation of coccolith is mainly regulated by changing internal CV pH .The
                                    pH of CV determines rate of deposition by influencing amount of inhibitors for
                                    coccolith deposition on nucleation sites. Bone mineralization was used as an
                                    inspiration for deposition part of the pathway which was implemented using
                                    differential equations governing nucleation, GPA formation, calcification rate and
                                    changing inhibitor concentration.</p>
                                <h3 id="section-0-subsection-0-2">Test</h3>
                                <p>When the external calcium concentration falls below a limit, the calcite deposition
                                    stops by reducing the pH inside the CV which corelates with literature. At 10 mM
                                    calcium concentration (sea water) value of calcification rate from this model
                                    corelates with observed value and optimal pH inside CV is also achieved. It is also
                                    shown that GPA and corresponding nucleation sites is not limiting rather uptake
                                    which is regulated by free calmodulin is limiting. Thereby improved calcification
                                    rates is shown by increasing amount of free calmodulin</p>
                                <h3 id="section-0-subsection-0-3">Build</h3>
                                <p>Deposition is not limiting rather uptake regulated by calmodulin is limiting step.
                                    External calcium concentration can't be increased beyond a point since if internal
                                    free cytosolic calcium increases due to it, will lead to cytotoxicity thus
                                    increasing amount of calmodulin expressed improves uptake as well as buffering -
                                    increasing overall calcification and carbon capture.</p>
                            </section>
                            <section id="section-1">
                                <h2>Population Model of Emilania Huxleyi</h2>
                                <hr>
                                <h3 id="section-1-subsection-1-0">Design</h3>
                                <p>We wanted to know what are the factors that are a problem to the growth and
                                    development of <i>Emilania Huxleyi</i> in the oceanic setting (particularly the
                                    region where the temperature for the growth is suitable). So we designed a
                                    population model (inspired by previous mesocosm and phytoplankton models) with full
                                    efforts to include all the factors that can impact the concerned coccolithophore.
                                </p>
                                <p>The model was designed to include ciliates, grazers, and other phytoplanktons (like
                                    diatoms, dinoflagellates, etc. that are prominent in the NE Atlantic). It also
                                    incorporates nutrients including nitrate, phosphate, ammonia, dissolved organic
                                    carbon, and silicates. Other necessary factors like the annual variation of
                                    irradiance and temperature were also included.</p>
                                <h3 id="section-1-subsection-1-1">Build</h3>
                                <p>Limitation due to nutrients, temperature and sunlight were incorporated using the
                                    Michaelis–Menten equation. Grazers were broadly characterized into ciliates, hetero
                                    flagellates and mesograzers along with bacteria. Which grazer eats which
                                    phytoplankton was properly taken care of. Variation of light intensity over the
                                    depth of ocean was taken care of using an exponentially decaying intensity function.
                                    Annual variation of mixed layer depth was input to the model. Similarly for the
                                    surface intensity and temperature. The mixing that happens between the mixed layer
                                    and the deep layer was modeled using the sinking of nutrients and organisms which is
                                    proportional to the difference in cell density. Finally free coccoliths and attached
                                    coccoliths were separately modeled.</p>
                                <h3 id="section-1-subsection-1-2">Test</h3>
                                <p>The Eulerian model provides results for all variables (population, net carbon
                                    capture, nutrient concentrations, etc.) as a function of time over one year.</p>
                                <p>We obtained population curves for <i>E. Huxleyi</i>, showing two blooms (for 47 N
                                    latitude), one large and one small, which aligns with actual blooming patterns. We
                                    adjusted nitrate concentrations and observed a significant impact on <i>E.
                                        Huxleyi</i>'s population. This level of improvement was not noticed for other
                                    organisms, demonstrating the critical nature of nitrate for <i>E. Huxleyi</i>'s
                                    growth.</p>
                                <p>Conversely, modifying phosphate concentrations did not result in a noticeable
                                    difference. Testing different half-saturation constants for nitrate showed a
                                    gradient of net carbon capture, offering a direction for organism modification. We
                                    also found that the nutrient concentration's effect on calcification rates per cell
                                    was not properly modeled, leading to unrealistic calcification rates compared to
                                    growth rates.</p>
                                <h3 id="section-1-subsection-1-3">Learn</h3>
                                <p>We learned that to make accurate qualitative and quantitative statements about carbon
                                    capture, we needed to correct the model regarding the nutrient impact on
                                    calcification rates. Supported by experimental data and other theoretical models, we
                                    re-modeled nutrient limitation using the Michaelis–Menten equation (initially,
                                    nutrient concentration only impacted calcification indirectly via cell density).</p>
                                <p>After modification, the ratio of carbon distribution between photosynthesis and
                                    calcification became consistent. We again calculated the gradient of net carbon
                                    capture concerning the half-saturation constant for nitrate, which remained
                                    significant.</p>
                                <h4>Key Equation</h4>
                                <p>The equation was adjusted by adding a multiplicative factor for nutrient limitation
                                    (phosphate and nitrate) in the first term.</p>
                            </section>
                            <section id="section-2">
                                <h2>PCR Amplification of Calmodulin</h2>
                                <hr>
                                <p>The gene of interest (Calmodulin) was amplified using Polymerase chain reaction
                                    (PCR). The gene of interest was synthesized from Integrated DNA Technologies (IDT).
                                    The appropriate annealing temperature was confirmed by the gradient PCR. The
                                    temperature with maximum amplification of the Calmodulin (CAM) gene was used to
                                    amplify the gene.</p>
                                <h4>Design</h4>
                                <p>The aim was to amplify our gene of interest, CAM, which would be used for cloning
                                    into the pET28a vector backbone.</p>
                                <h4>Build</h4>
                                <p>The gradient PCR from 47 - 65 ℃ was performed using TaKaRa PrimeSTAR® Max DNA
                                    Polymerase, with the parameters according to the manufacturer’s protocol. The
                                    synthesized CAM received from IDT was used as a template, and primers were used at a
                                    concentration of 0.5 µM. The PCR products obtained from gradient PCR were analyzed
                                    on a 1 % agarose gel following the standard protocol. The temperature yielding the
                                    best amplification was selected for the next round of PCR using the same set of
                                    primers for downstream applications. </p>
                                <h4>Test</h4>
                                <p>The PCR amplification was observed for the CAM gene (~500 bp) corresponding to the
                                    reference 1Kb DNA ladder (GeneRuler 1kb DNA ladder,Thermo Scientific) with no
                                    non-specific bands, confirming the accuracy of the amplification process.</p>
                                <h4>Learn</h4>
                                <p>The gene product was successfully amplified at an optimum annealing temp of 55°C. The
                                    amplified product can be used in the downstream cloning procedure. </p>
                                <figure><img src="https://static.igem.wiki/teams/5365/registry-parts/picture3.png"
                                        alt=""></figure>
                            </section>
                            <section id="section-3">
                                <h2>Cloning of Calmodulin in pET28a</h2>
                                <hr>
                                <h3 id="section-3-subsection-3-0">Cycle 1: Restriction digestion</h3>
                                <h4>Design</h4>
                                <p>The aim was to clone our gene of interest, CAM, into the pET28a vector, following its
                                    amplification. To ensure the successful insertion of the CAM gene in pET28a, NcoI,
                                    and HindIII restriction sites were inserted at the 5’ end and 3’ end with the help
                                    of forward and reverse primers respectively. The amplified product and pET28a were
                                    digested with NcoI and HindIII. The selection of restriction sites allows the
                                    presence of 6x His Tag at the C-terminal of recombinant protein and would help in
                                    protein purification. The tag allows the binding of recombinant protein to the
                                    Ni-NTA complex conjugated to Sepharose beads.</p>
                                <h4>Build</h4>
                                <p>The pET28a and the amplified CAM were digested with restriction enzymes, NcoI and
                                    HindIII in two separate reactions and incubated for 1.5 hours at 37 °C. The
                                    digestion condition was according to the manufacturer protocol (refer Thermo). After
                                    digestion, the double-digested products were purified and ligated using the T4
                                    ligase enzyme. The ligated product was transformed into XL1-blue competent cells.
                                    XL1-blue cells would act as plasmid maintenance strain. Five colonies were
                                    inoculated into fresh LB broth media along with proper anti-biotics which would help
                                    in the growth of the transformed colonies. Plasmid was isolated from the inoculated
                                    culture and tested for insert release using double digestion.</p>
                                <h4>Test</h4>
                                <p>After the restriction digestion to check the insert release, 1% agarose gel was run.
                                    The expected fragment sizes were ~5.4 kb for the linearized vector and ~500 bp for
                                    the insert. Also, ten more colonies were screened for the presence of the CAM gene
                                    using PCR amplification using appropriate primers and individual colonies.</p>
                                <h4>Learn</h4>
                                <p>Only a single band corresponding to the vector size was observed. No bands were seen
                                    at the expected size as of insert in either of the five colonies used to check for
                                    positive colonies. Also, no amplification was observed corresponding to the insert
                                    length size. We speculate that the reason for the no detection of the insert, either
                                    by double digestion or colony PCR could be partial or improper digestion and due to
                                    which vector backbone is self-ligating and getting selected on antibiotic selection
                                    plates. So we thought of proceeding with a different cloning strategy.</p>
                                <figure><img src="https://static.igem.wiki/teams/5365/registry-parts/picture10.png"
                                        alt=""></figure>
                                <h3 id="section-3-subsection-3-1">Cycle 2: Cloning of Calmodulin gene using Infusion Kit
                                </h3>
                                <h4>Design</h4>
                                <p>Since the cloning strategy using double-digestion did not yield any positive clones.
                                    Therefore, we proceeded with a more efficient approach, infusion cloning. The
                                    strategy includes linearizing the plasmid vector and amplifying the primers with
                                    homologous sequences to the ends of the linearized vector. Then the incubation of
                                    linearized vector and amplified PCR product with the required amount of the reaction
                                    buffer provided with the kit. The benefit of using an infusion cloning kit over
                                    other methods is that it requires only 10-15 bp homologous sequences in primers for
                                    PCR as compared to 30-40 bp in other similar cloning methods.</p>
                                <h4>Build</h4>
                                <p>For In-fusion cloning( In-FusionⓇ HD cloning kit, TaKaRa bio), we linearized the
                                    pET28a vector with NcoI and HindIII and also amplified the Calmodulin gene with
                                    primers containing the homologous sequences to the ends of the linearized vector.
                                    The 20µL reaction mixture was prepared using the reaction buffer provided with the
                                    kit and the reaction mixture was incubated at 500C for 30 minutes. The reaction
                                    mixture was used to transform into the XL1-blue cells and the colonies were
                                    inoculated into the LB broth media and plasmids were isolated and checked for the
                                    presence of the Calmodulin gene using PCR amplification, using plasmids as template
                                    and appropriate primers.</p>
                                <h4>Test</h4>
                                <p>PCR-amplified products were run on 1% agarose gel to identify the true recombinant
                                    plasmids. We were able to observe an amplified band adjacent to the size of the
                                    insert (Calmodulin gene) for each of the five isolated plasmids from different
                                    colonies. Additionally, we got the plasmid sequenced (from Eurofins Scientific) to
                                    check for any point mutation or frameshift mutation and also to ensure the accurate
                                    insertion and correct orientation of the insert.</p>
                                <h4>Learn</h4>
                                <p>The Calmodulin gene was successfully cloned in a plasmid vector and the sequence was
                                    confirmed using Sanger sequencing of the cloned plasmid. The true recombinant
                                    plasmid was used for the transformation of the expression strain of E. coli
                                    (BL21-DE3 RIPL).</p>
                                <figure><img src="https://static.igem.wiki/teams/5365/registry-parts/picture5.png"
                                        alt=""></figure>
                            </section>
                            <section id="section-4">
                                <h2>Check the expression of the Calmodulin protein </h2>
                                <hr>
                                <h3 id="section-4-subsection-4-0">Cycle 1: Expression in BL21-DE3 RIPL cells</h3>
                                <p>BL21-CodonPlus strains are engineered to contain extra copies of genes that encode
                                    the tRNAs that most frequently limit the translation of heterologous proteins in E.
                                    coli. They are ideal for difficult protein expression, especially when codon bias is
                                    a problem. RIPL stands for Arginine, Isoleucine, Proline, and Leucine.</p>
                                <h4>Design</h4>
                                <p>The plasmid confirmed to contain the correctly inserted Calmodulin gene was used to
                                    transform BL21-DE3 RIPL competent cells by heat shock method and plated over the
                                    antibiotic selection plate.</p>
                                <h4>Build</h4>
                                <p>BL21-DE3 RIPL competent cells were freshly prepared and were used for the
                                    transformation. BL21-DE3 RIPL cells have antibiotic resistance against
                                    Chloramphenicol while pET28a has resistance against Kanamycin. 50ng of plasmid DNA
                                    was used for transformation by heat shock method and after recovery, the pellet was
                                    resuspended in resuspended in 100µL media. 50 µL of this suspension was plated over
                                    LB agar plates with Chloramphenicol and Kanamycin. Three different colonies were
                                    picked and inoculated into LB broth media with desired antibiotics added to it. 1%
                                    of primary inoculum was added to 50mL LB broth media and after growth at OD600 of
                                    0.6, the culture was induced with 0.5mM IPTG and incubated at 300C for 4 hours.
                                    Culture was pelleted down by centrifugation at 6000xg for 10 minutes. Supernatant
                                    was discarded and the pellet was resuspended in 5mL of lysis buffer. The bacterial
                                    lysate was sonicated at 40% amplitude and pulse of 2 sec ON 2 sec OFF and total time
                                    of 5 minutes. The lysate was centrifuged at 14000xg to separate the supernatant
                                    containing solubilized protein and the pellet contained mostly membrane and
                                    insoluble protein.</p>
                                <h4>Test</h4>
                                <p>After transformation, the colonies were observed on LB agar plates with the
                                    antibiotics used for the selection of the true transformant. SDS-Page gel was run to
                                    visualize the recombinant protein. We were not able to see any distinct band
                                    corresponding to the size of the Calmodulin protein in the lane with induced lysate
                                    loaded as compared to the uninduced. The standard ladder with a range from 10kDa to
                                    180kDa was run, acting as a reference for the molecular size of the protein.</p>
                                <h4>Learn</h4>
                                <p>As there was no distinct band corresponding to the size of the target protein, the
                                    reason could be either low expression or no expression. To test the low expression,
                                    we thought to proceed with the western blotting in the next cycle.</p>
                                <figure><img src="https://static.igem.wiki/teams/5365/registry-parts/picture11.png"
                                        alt=""></figure>
                                <h3 id="section-4-subsection-4-1">Cycle 2: Western blot to check the Expression in
                                    BL21-DE3 RIPL cells</h3>
                                <p>A Western blot, also known as protein immunoblot, is a laboratory technique used to
                                    detect specific proteins in a complex mixture of proteins.</p>
                                <h4>Design</h4>
                                <p>Proteins are separated by size using gel electrophoresis (SDS-PAGE). The separated
                                    proteins are transferred (blotted) onto a membrane (PVDF) using an electric current.
                                    The membrane is exposed to an antibody specifically designed to recognize and bind
                                    to the target protein. The bound antibody is detected using a secondary antibody
                                    labeled with a chromogenic probe.</p>
                                <h4>Build</h4>
                                <p>The SDS-Page gel was run using the samples prepared in the previous step. The gel was
                                    washed with water and the transfer assembly was made in the following series-
                                    Cathode-Blotting sheet- Gel- PVDF-Blotting sheet- Anode. The transfer of protein
                                    bands from gel to PVDF membrane was carried at a constant current of 250mA for 30
                                    minutes. The PVDF membrane was incubated with blocking solution for 2 hours washed 3
                                    times and then incubated with primary antibody (anti-His antibody) for 2 hours and
                                    then again washed and then finally incubated with secondary antibody for 1 hour. The
                                    secondary antibody in this case is HRP conjugated.</p>
                                <h4>Test</h4>
                                <p>To check the presence of the target protein which contains 6x His tag at its
                                    C-terminal, the primary antibody used would be able to bind to the 6x His peptide,
                                    and the secondary antibody is HRP conjugated. The blot was developed by applying the
                                    solution of peroxide and luminol as a chemiluminescence method. However, after
                                    development, no bands were observed in the entire blot sheet whereas the control
                                    lane showed a highly enhanced signal. This confirms that there was no expression of
                                    Calmodulin protein in BL21-DE3 RIPL cells.</p>
                                <h4>Learn</h4>
                                <p>To check the presence of the target protein which contains 6x His tag at its
                                    C-terminal, the primary antibody used would be able to bind to the 6x His peptide,
                                    and the secondary antibody is HRP conjugated. The blot was developed by applying the
                                    solution of peroxide and luminol as a chemiluminescence method. However, after
                                    development, no bands were observed in the entire blot sheet whereas the control
                                    lane showed a highly enhanced signal. This confirms that there was no expression of
                                    Calmodulin protein in BL21-DE3 RIPL cells.</p>
                                <figure><img src="https://static.igem.wiki/teams/5365/registry-parts/picture7.png"
                                        alt=""></figure>
                                <h3 id="section-4-subsection-4-2">Cycle 1: Expression in BL21-DE3 Rosetta cells</h3>
                                <p>As the target protein is of eukaryotic expression we also thought of using BL21(DE3)
                                    Rosetta cells to check the expression of the Calmodulin protein. Rosetta™ host
                                    strains are BL21 derivatives designed to enhance the expression of eukaryotic
                                    proteins that contain codons rarely used in E. coli. These strains supply tRNAs for
                                    AGG, AGA, AUA, CUA, CCC, and GGA codons on a compatible chloramphenicol-resistant
                                    plasmid.</p>
                                <h4>Design</h4>
                                <p>The plasmid confirmed to contain the correctly inserted Calmodulin gene was used to
                                    transform BL21-DE3 Rosetta competent cells by heat shock method and plated over the
                                    antibiotic selection plate.</p>
                                <h4>Build</h4>
                                <p>BL21-DE3 Rosetta competent cells were freshly prepared and were used for the
                                    transformation. BL21-DE3 Rosetta cells have antibiotic resistance against
                                    Chloramphenicol while pET28a have resistance against Kanamycin. 50ng of plasmid DNA
                                    was used for transformation by heat shock method and after recovery, the pellet was
                                    resuspended in resuspended in 100µL media. 50 µL of this suspension was plated over
                                    LB agar plates with Chloramphenicol and Kanamycin. Three different colonies were
                                    picked and inoculated into LB broth media with desired antibiotics added to it. 1%
                                    of primary inoculum was added to 50mL LB broth media and after growth at OD600 of
                                    0.6, the culture was induced with 0.5mM IPTG and incubated at 300C for 4 hours. The
                                    culture was pelleted down by centrifugation at 6000xg for 10 minutes. The
                                    supernatant was discarded and the pellet was resuspended in 5mL of lysis buffer. The
                                    bacterial lysate was sonicated at 40% amplitude and pulse of 2 sec ON 2 sec OFF and
                                    a total time of 5 minutes. The lysate was centrifuged at 14000xg to separate the
                                    supernatant containing solubilized protein and the pellet contained mostly membrane
                                    and insoluble protein.</p>
                                <h4>Test</h4>
                                <p>After transformation, the colonies were observed on LB agar plates with the
                                    antibiotics used for the selection of the true transformant. SDS-Page gel was run to
                                    visualize the recombinant protein. We were not able to see any distinct band
                                    corresponding to the size of the Calmodulin protein in the lane with induced lysate
                                    loaded as compared to the uninduced. The standard ladder with a range from 10kDa to
                                    180kDa was run, acting as a reference for the molecular size of the protein.</p>
                                <h4>Learn</h4>
                                <p>As there was no distinct band corresponding to the size of the target protein, the
                                    reason could be either low expression or no expression. To test the low expression,
                                    we thought to proceed with the western blotting in the next cycle.</p>
                                <figure><img src="https://static.igem.wiki/teams/5365/registry-parts/picture8.png"
                                        alt=""></figure>
                                <h3 id="section-4-subsection-4-3">Cycle 2: Western blot to check the Expression in
                                    BL21-DE3 Rosetta cells</h3>
                                <p>A Western blot, also known as protein immunoblot, is a laboratory technique used to
                                    detect specific proteins in a complex mixture of proteins.</p>
                                <h4>Design</h4>
                                <p>Proteins are separated by size using gel electrophoresis (SDS-PAGE). The separated
                                    proteins are transferred (blotted) onto a membrane (PVDF) using an electric current.
                                    The membrane is exposed to an antibody specifically designed to recognize and bind
                                    to the target protein. The bound antibody is detected using a secondary antibody
                                    labeled with a chromogenic probe.</p>
                                <h4>Build</h4>
                                <p>The SDS-Page gel was run using the samples prepared in the previous step. The gel was
                                    washed with water and the transfer assembly was made in the following series-
                                    Cathode-Blotting sheet- Gel- PVDF-Blotting sheet- Anode. The transfer of protein
                                    bands from gel to PVDF membrane was carried at a constant current of 250mA for 30
                                    minutes. The PVDF membrane was incubated with blocking solution for 2 hours washed 3
                                    times and then incubated with primary antibody (anti-His antibody) for 2 hours and
                                    then again washed and then finally incubated with secondary antibody for 1 hour. The
                                    secondary antibody in this case is HRP conjugated.</p>
                                <h4>Test</h4>
                                <p>To check the presence of the target protein which contains 6x His tag at its
                                    C-terminal, the primary antibody used would be able to bind to the 6x His peptide,
                                    and the secondary antibody is HRP conjugated. The blot was developed by applying the
                                    solution of peroxide and luminol as a chemiluminescence method. However, after
                                    development, no bands were observed in the entire blot sheet whereas the control
                                    lane showed a highly enhanced signal. This confirms that there was no expression of
                                    Calmodulin protein in BL21-DE3 Rosetta cells.</p>
                                <h4>Learn</h4>
                                <p>The reason for the no expression of Calmodulin even in the Rosetta cells cannot be
                                    explained. However, some other modifications in the expression check design can be
                                    done. Also, it is possible that these cells are somehow losing the recombinant
                                    plasmid.</p>
                                <figure><img src="https://static.igem.wiki/teams/5365/registry-parts/picture9.png"
                                        alt=""></figure>
                            </section>
                            <p>WE HAVE PLANNED THE FOLLOWING CYCLES TO BE PERFORMED AS PROTEIN GETS EXPRESSED.</p>
                            <section id="section-5">
                                <h2>Purification of protein from large culture volume</h2>
                                <hr>
                                <h3 id="section-5-subsection-5-0">Affinity purification</h3>
                                <h4>Design</h4>
                                <p>Affinity purification, also known as affinity chromatography, is a powerful technique
                                    used to separate and purify specific proteins or protein complexes from complex
                                    mixtures. It relies on the specific and reversible binding of a protein to a
                                    matrix-bound ligand.</p>
                                <h4>Build</h4>
                                <p>For large volume culture, 1 liter of LB broth media shall be inoculated using 1%
                                    inoculum from overnight grown primary culture. The culture shall be grown at 370C
                                    with constant shaking at 150 RPM and induced once the OD600 of the culture reaches
                                    0.6. After induction, the cells shall be lysed to release soluble proteins. Then the
                                    lysate shall be passed over a nickel-nitrilotriacetic acid (Ni-NTA) resin, which
                                    will specifically bind the histidine-tagged protein while allowing impurities to
                                    wash away. The target protein shall be eluted using imidazole, which shall compete
                                    with the histidine residues for binding.</p>
                                <h4>Test</h4>
                                <p>After the restriction digestion to check the insert release, 1% agarose gel was run.
                                    The expected fragment sizes were ~5.4 kb for the linearized vector and ~500 bp for
                                    the insert. Also, ten more colonies were screened for the presence of the CAM gene
                                    using PCR amplification using appropriate primers and individual colonies.</p>
                                <h4>Learn</h4>
                                <p>Only a single band corresponding to the vector size was observed. No bands were seen
                                    at the expected size as of insert in either of the five colonies used to check for
                                    positive colonies. Also, no amplification was observed corresponding to the insert
                                    length size. We speculate that the reason for the no detection of the insert, either
                                    by double digestion or colony PCR could be partial or improper digestion and due to
                                    which vector backbone is self-ligating and getting selected on antibiotic selection
                                    plates. So we thought of proceeding with a different cloning strategy.</p>
                                <h3 id="section-5-subsection-5-1">Size exclusion chromatography purification</h3>
                                <p>Size Exclusion Chromatography (SEC) is a chromatographic technique used to separate
                                    molecules based on their size. It works by filtering a mixture through a gel or
                                    porous beads with pores of a specific size distribution. The gel or beads act as a
                                    molecular sieve, allowing smaller molecules to pass through while larger molecules
                                    are excluded.</p>
                                <h4>Design</h4>
                                <p>Size exclusion chromatography (SEC) is a technique used to separate proteins and
                                    other macromolecules based on their size.</p>
                                <h4>Build</h4>
                                <p>The partially purified protein after affinity purification shall be concentrated and
                                    loaded upon the SEC column. Samples are passed through a column filled with porous
                                    beads, allowing smaller molecules to enter the pores and thus travel more slowly,
                                    while larger molecules bypass the beads and elute faster. SEC is often employed as a
                                    final purification step to achieve higher purity and to analyze the size and
                                    homogeneity of protein samples.</p>
                            </section>
                            <section id="section-6">
                                <h2>Biophysical Characterization of Purified Calmodulin</h2>
                                <hr>
                                <h3 id="section-6-subsection-6-0">I) CD of Calmodulin</h3>
                                <h4>Design</h4>
                                <p>As Calmodulin protein interacts with Ca2+, Circular Dichroism (CD) is a spectroscopic
                                    technique used to study the secondary structure of proteins and other chiral
                                    molecules by measuring the differential absorption of left- and right-handed
                                    circularly polarized light. It provides insights into protein folding,
                                    conformational changes, and interactions, making it a valuable tool in biochemistry
                                    and biophysics.</p>
                                <h3 id="section-6-subsection-6-1">II) Fluorescence</h3>
                                <h4>Design</h4>
                                <p>The fluorescence of proteins at 280 nm primarily arises from the aromatic amino
                                    acids, particularly tryptophan, tyrosine, and phenylalanine. When proteins are
                                    excited with ultraviolet light around this wavelength, these aromatic residues
                                    absorb the energy and emit fluorescence, typically in the range of 300 to 350 nm.
                                </p>
                                <h4>Build</h4>
                                <p>We shall formulate a solution containing the protein of interest with different
                                    calcium ion concentrations. Measuring fluorescence at 280 nm shall be used in
                                    protein studies for assessing conformational changes and interactions.</p>
                            </section>
                            <section id="section-7">
                                <h2>Biochemical Characterization of Calmodulin</h2>
                                <hr>
                                <h3 id="section-7-subsection-7-0">Check the affinity of Calmodulin with Ca2+ ions</h3>
                                <h4>Design</h4>
                                <p>Calmodulin protein is known to interact with Ca2+ ions. To check this interaction, we
                                    designed a matrix-based column that would bind to Ca2+. When the purified protein is
                                    passed through the matrix, it binds to the matrix and elutes using a buffer with a
                                    high concentration of Ca2+ ions.</p>
                                <h4>Build</h4>
                                <p>Calmodulin will be purified from E. coli after expressing it as a recombinant
                                    protein. Initially, it will be dissolved in a calcium-containing buffer to maintain
                                    its active form. Phenothiazine analog 2-chloro-10-(3-aminopropyl) phenothiazine
                                    (CAPP) needs to be coupled to Sepharose 4B beads using cyanogen bromide as an
                                    activating agent. This immobilization process must be carried out under specific
                                    conditions to ensure the stability of the phenothiazine and the efficiency of the
                                    coupling. The beads will be protected from light and kept at low temperatures (4°C)
                                    due to the light-sensitive nature of phenothiazines.</p>
                                <figure><img
                                        src="https://static.igem.wiki/teams/5365/dbtl/whatsapp-image-2024-10-02-at-00-11-28.jpeg"
                                        alt=""></figure>
                                <figure><img
                                        src="https://static.igem.wiki/teams/5365/dbtl/whatsapp-image-2024-10-02-at-00-11-28-1.jpeg"
                                        alt=""></figure>
                            </section>
                            <section id="section-8">
                                <h2>NAD+ Kinase Assay</h2>
                                <hr>
                                <h3 id="section-8-subsection-8-0">Design</h3>
                                <p>Calmodulin (CAM), our protein of interest, is responsible for the transport of Ca2+
                                    ions at the site of calcification, i.e., the coccolith vesicle in E-Huxleyi. The
                                    objective of our assay is to verify the activity of CAM purified from E. coli as a
                                    recombinant protein derived from Emilania Huxleyi. NADK (NAD Kinase) is an enzyme
                                    responsible for the phosphorylation of NAD⁺ (nicotinamide adenine dinucleotide) to
                                    form NADP⁺ (nicotinamide adenine dinucleotide phosphate). NADK is activated through
                                    Ca²⁺-Ca²-dependent CAM signaling, enhancing the production of NADP⁺ and increasing
                                    the availability of NADPH.</p>
                            </section>
                            <section id="section-9">
                                <h2>Cloning of Calmodulin with C-terminal Flag Tag</h2>
                                <hr>
                                <h3 id="section-9-subsection-9-0">Design</h3>
                                <p>Since affinity purification using a His-Tag is not as efficient as a Flag tag, we
                                    thought of cloning the Calmodulin gene with a Flag-tag at the C-terminal for
                                    co-immunoprecipitation experiments.</p>
                                <h4>Build</h4>
                                <p>The pET28a vector was linearized with NcoI and HindIII, and the Calmodulin gene was
                                    amplified with primers containing homologous sequences to the ends of the linearized
                                    vector. The Flag tag was introduced using the reverse primer, and the stop codon was
                                    included after the Flag tag sequence. The reaction mixture was used to transform
                                    into the XL1-blue cells, and the colonies were screened for the presence of the
                                    Calmodulin gene using PCR amplification.</p>
                            </section>
                            <section id="section-10">
                                <h2>Co-Immunoprecipitation of Interacting Proteins from E. huxleyi</h2>
                                <hr>
                                <h3 id="section-10-subsection-10-0">Design</h3>
                                <p>The co-immunoprecipitation (Co-IP) procedure using a Flag tag involves lysing cells
                                    expressing the Flag-tagged protein and incubating the lysate with anti-flag
                                    antibodies immobilized on a resin to capture the protein and its interacting
                                    partners. The protein complex can then be eluted and analyzed through techniques
                                    like SDS-PAGE and Western blotting to study protein interactions.</p>
                            </section>
                        </div>
                    </div>
                </div>
            </main>
        </div>
        <footer class="pt-3 pb-3 footer  text-white" >
            <div class="container" >
              <div class="row mb-3 align-items-center">
                <div class="col-lg-12 col-xs-12">
                  <div class="logo-section text-center"><img width="15%"
                      src="https://static.igem.wiki/teams/5365/igem-logo.png" class="img-fluid mb-3" alt="iGEM IIT-B Logo">
                      <div class="sponsor-logos d-flex justify-content-between align-items-center"><img
                        src="https://static.igem.wiki/teams/5365/wiki-assets/1.png" alt="Sponsor 1" class="sponsor-logo" >
                        <img
                        src="https://static.igem.wiki/teams/5365/wiki-assets/2.png" alt="Sponsor 2" class="sponsor-logo">
                        <img
                        src="https://static.igem.wiki/teams/5365/wiki-assets/3.png" alt="Sponsor 3" class="sponsor-logo">
                        <img
                        src="https://static.igem.wiki/teams/5365/wiki-assets/4.png" alt="Sponsor 4" class="sponsor-logo">
                        <img
                        src="https://static.igem.wiki/teams/5365/wiki-assets/wrcb-new-wf-logo-2024.png" alt="Sponsor 5"
                        class="sponsor-logo"></div>
                  </div>
        
                </div>
              </div>
              <hr>
              <div class="row mt-3 justify-content-center align-items-center">
                <div class="col-lg-4 col-md-6 col-xs-12 text-center">
                  <h5 class="mt-0" style="color: black;">Social Media</h5>
                  <ul class="list-inline m-0">
                    <li class="list-inline-item" style="color: black;"><a href="https://www.instagram.com/igem_iitb" target="_blank"
                        rel="noreferrer">Instagram</a></li>
                    <li class="list-inline-item" style="color: black;"><a href="https://www.linkedin.com/company/igem-iit-bombay" target="_blank"
                        rel="noreferrer">LinkedIn</a></li>
                  </ul>
                </div>
                <div class="col-lg-4 col-md-6 col-xs-12 text-center">
                  <h5 class="mt-0" style="color: black;">Contact</h5>
                  <p class="mb-1" style="color: black;">IIT Bombay,
                    Powai,
                    Mumbai,
                    Maharashtra,
                    400076</p>
                  <p class="mb-0" style="color: black;">igem.iitb@gmail.com</p>
                </div>
              </div>
              <div class="row mt-3">
                <div class="col text-center">
                  <p class="mb-1" style="color: black;"><small>© 2024 - Content on this site is licensed under a <a class="subfoot"
                        href="https://creativecommons.org/licenses/by/4.0/" rel="license">Creative Commons Attribution 4.0
                        International license</a>.</small></p>
                  <p class="mb-0" style="color: black;"><small>The repository used to create this website is available at<a
                        href="https://gitlab.igem.org/2024/iit-bombay">gitlab.igem.org/2024/iit-bombay</a>.</small></p>
                </div>
              </div>
            </div>
          </footer>
    </div>


</body>
<script>
    // Get all dropdown elements on the page
    const dropdowns = document.querySelectorAll('.dropdown');
    
    // Add click event listener to each dropdown
    dropdowns.forEach(dropdown => {
        const tabLink = dropdown.querySelector('.tab-link');
        const dropdownMenu = dropdown.querySelector('.dropdown-menu');
        
        // Toggle dropdown when clicking the tab link
        tabLink.addEventListener('click', () => {
            if (dropdownMenu.style.display === 'none' || dropdownMenu.style.display === '') {
                dropdownMenu.style.display = 'block';
            } else {
                dropdownMenu.style.display = 'none';
            }
        });
        
        // Close dropdown when clicking outside
        document.addEventListener('click', (event) => {
            if (!dropdown.contains(event.target)) {
                dropdownMenu.style.display = 'none';
            }
        });
    });
</script>
</html>
